speech/tests/system.py

```diff
@@ -245,3 +247,37 @@ def test_stream_recognize_single_utterance(self):
             for results in self._make_streaming_request(
                     file_obj, single_utterance=False):
                 self._check_results(results.alternatives)
+
+    def test_long_running_recognize(self):
+        content = None
+        repetitions = 5
+        with open(AUDIO_FILE, 'rb') as in_stream:
+            in_wavfile = wave.open(in_stream)
+            params = in_wavfile.getparams()
+            frames = in_wavfile.readframes(in_wavfile.getnframes())
+
+        with io.BytesIO() as out_stream:
+            out_wavfile = wave.open(out_stream, 'w')
+            out_wavfile.setparams(params)
+            for _ in range(repetitions):
+                out_wavfile.writeframes(frames)
+                content = out_stream.getvalue()
+
+        client = speech.SpeechClient()
+
+        operation = client.long_running_recognize(
+            config=speech.types.RecognitionConfig(
+                encoding='LINEAR16',
+                language_code='en-US',
+            ),
+            audio=speech.types.RecognitionAudio(content=content)
+        )
+
+        op_result = operation.result()
+        wav_transcription = "hello thank you for using google cloud platform "
+        self.assertGreater(len(op_result.results), 0)
+        for result in op_result.results:
+            self.assertGreater(len(result.alternatives), 0)
+            for alternative in result.alternatives:
+                self.assertEqual(alternative.transcript.lower(),
+                                 (wav_transcription * repetitions)[:-1])
```

Comment:
@chemelnucfin Why the use of `[:-1]`?

Reply:
remove the final space on the transcription.  Should I comment it or make it more clear somehow?

Reply:
Ah I see. Instead I suggest the following.

```python
wav_text = 'hello thank you for using google cloud platform'
responses = [wav_text] * repetitions
transcript = ' '.join(responses)
...
for result in op_result.results:
    self.assertGreater(len(result.alternatives), 0)
    for alternative in result.alternatives:
        self.assertEqual(
            alternative.transcript.lower(), transcript)
```

----

Also, sorry for not mentioning it before but please use single quotes, not double quotes, for string variables (you used double quotes in `wav_transcription`). @tseaver has a bit of nuance for this rule when it comes to exception text, but let's cross that bridge when we get there.

Reply:
I made it more clear what I was doing in the new commit.  thanks.

Reply:
got it.